qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -775,3 +773,13 @@ void HOCRPdfExporter::updateValid() {
 	}
 	ui.buttonBox->button(QDialogButtonBox::Ok)->setEnabled(valid);
 }
+
+void HOCRPdfExporter::addInfoClicked() {
+	auto source = MAIN->getSourceManager()->getSelectedSources().first();
```

Comment:
Personal style preference: if the type isn't some long templated thing, just spell it out (makes things more readable).

Rest looks good.

Reply:
Fixed.